.eslintrc.js

```diff
@@ -42,6 +43,16 @@ module.exports = {
     'react/jsx-filename-extension': 'off',
     'react/jsx-props-no-spreading': 'off',
     'react/require-default-props': 'off',
+    'styled-components-a11y/label-has-for': [
```

Comment:
force labels to have an id is important to connect them with input/textfields for screen reader

Reply:
Please move this to the right place, these are all rules that are set to `off`. Is this rule disabled per default?